docs/plans/volunteer-management-plan.md

```
@@ -0,0 +1,1019 @@
+    with: {
+      capabilities: {
+        where: eq(competitionVolunteerRoleCapabilitiesTable.capability, "judge"),
+      },
+    },
+  }).then(roles => roles.filter(r => r.capabilities.length > 0));
+}
+```
+
+### 1.2 Organizer UI (`/compete/organizer/[id]/volunteers`)
+- **Settings/Roles Tab**:
+    - CRUD interface for `Volunteer Roles`.
```

markdownlint-cli2 (0.18.1)

Unordered list indentation
Expected: 2; Actual: 4

(MD007, ul-indent)